src/GitHubCreateMergePRs/config.xml

```diff
@@ -11,25 +11,22 @@
     <merge from="release/dev16.6" to="master" />
```

Comment:
I think at this point we can flow 16.6 into 16.7. We were only trying to avoid merging into 16.7-p2